src/Instance/HoldingMovementList/HoldingMovement/HoldingMovement.js

```diff
@@ -0,0 +1,139 @@
+import React, {
+  useMemo,
+  useCallback,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  IfPermission,
+} from '@folio/stripes/core';
+import {
+  Accordion,
+  Row,
+  Col,
+  Button,
+  Checkbox,
+} from '@folio/stripes/components';
+
+import {
+  callNumberLabel
+} from '../../../utils';
+import { ItemsListContainer } from '../../ItemsList';
+
+const HoldingMovement = ({
+  holding,
+  referenceData,
+  onViewHolding,
+  onAddItem,
+
+  draggable,
+  droppable,
+  ifItemsDragSelected,
+  selectItemsForDrag,
+  selectHoldingsForDrag,
+  ifHoldingDragSelected,
+  getDraggingItems,
+}) => {
+  const { locationsById } = referenceData;
+  const labelLocation = holding.permanentLocationId ? locationsById[holding.permanentLocationId].name : '';
+
+  const viewHoldings = useCallback(() => {
+    onViewHolding();
+  }, [onViewHolding]);
+
+  const addItem = useCallback(() => {
+    onAddItem();
+  }, [onAddItem]);
+
+  const holdingButtonsGroup = useMemo(() => {
+    return (
+      <>
+        <Button
+          id={`clickable-view-holdings-${holding.id}`}
+          data-test-view-holdings
+          onClick={viewHoldings}
+        >
+          <FormattedMessage id="ui-inventory.viewHoldings" />
+        </Button>
+
+        <IfPermission perm="ui-inventory.item.create">
+          <Button
+            id={`clickable-new-item-${holding.id}`}
+            data-test-add-item
+            onClick={addItem}
+            buttonStyle="primary paneHeaderNewButton"
+          >
+            <FormattedMessage id="ui-inventory.addItem" />
+          </Button>
+        </IfPermission>
+      </>
+    );
+  }, [holding.id, viewHoldings, addItem]);
+
+  return (
+    <div>
+      <FormattedMessage id="ui-inventory.moveItems.selectItem">
```

Comment:
IMHO, render output is much readable if you use `useIntl()` and intl.formatMessage for cases like that, when you need particularly a string value